packages/itmat-ui-react/src/components/users/createNewUser.tsx

```diff
@@ -1,16 +1,15 @@
 import * as React from 'react';
-import { useMutation, ExecutionResult } from 'react-apollo';
-import { NavLink, Redirect } from 'react-router-dom';
-import { CREATE_USER, GET_USERS } from 'itmat-commons/dist/graphql/appUsers';
+import { useMutation } from 'react-apollo';
+import { NavLink } from 'react-router-dom';
+import { CREATE_USER } from 'itmat-commons/dist/graphql/user';
 import css from './userList.module.css';
 
 export const CreateNewUser: React.FunctionComponent = () => {
-    const [completedCreationId, setCompletedCreationId] = React.useState(undefined);
+    const [completedCreationId, setCompletedCreationId] = React.useState(false);
```

Comment:
if you change the state type, then change the variable name to reflect that

Reply:
ok :)